frontend/src/routes/ContentWraper/MainBox/CardWrapper.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box, Center, Container, Flex, Heading, Image, Spacer, Wrap, WrapItem } from "@chakra-ui/react";
+import { useWalletService } from "@services/WalletService";
+import { ReactElement } from "react";
+import { AwardCard } from "./AwardCard";
+
+export const CardWrapper = (): ReactElement => {
+    return (
+        <>
+            <Container maxW="150ch" centerContent>
+                <Wrap spacing='70px' justify='center'>
+                    <WrapItem>
+                        <Center w='300px' h='300px'>
```

Comment:
I think w and h are a problem with clipped edges.

Reply:
I added a spacer at the bottom and it solved this issue